dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -243,8 +243,10 @@ DialogYesButton=Yes
 DialogNoButton=No
 DialogCancelButton=Cancel
 
-ProjectNotImportedDialogTitle=The {0} project is not imported into Eclipse
-ProjectNotImportedDialogMsg=There is no source available for debugging because the project has not been imported. Import the {0} project?
+ProjectMustBeOpenedToDebugTitle=Project must be opened
+ProjectMustBeOpenedToDebugMessage=To perform the selected action, the project must first be opened in a window. Continue?
```

Comment:
Will the program open the project for the user? If so, maybe, "To perform the selected action, Codewind opens the project in a window. Continue?"

If the user has to perform the action: "To perform the selected action, open the project in a window. Continue?"